### DISCUSSION TITLE
[Announcement] Final exam is graded

### DISCUSSION THREAD

**Discussion by goodtaeeun** [prosyslab-classroom/cs524-program-analysis#447] [Announcements]
[English version follows]

안녕하세요, 기말고사 채점이 완료되었습니다.
Gradescope에서 점수와 채점 사유를 확인하실 수 있습니다.
KLMS에도 성적이 게시되었으니 16주차의 "Final" 과제를 확인하시기 바랍니다.

성적에 이의가 있으시다면 Gradescope의 재채점 요청 기능을 사용하시거나 cs524.ta@prosys.kr로 이메일을 주시길 바랍니다.
이의 신청은 이번주 금요일, 6월 14일 밤 11시 59분까지 받도록 하겠습니다.

참고로 이번 시험의 평균은 37, 표준편차는 22입니다.

---

Hi all,
The final exam is graded.

You can check your score and criteria for each problem at Gradescope.
The scores are also uploaded on KLMS. Check the assignment named "Final" on week 16.
If you want to make any claims for your score, send a "Regrade request(재채점 요청)" in the Gradescope system, or an email cs524.ta@prosys.kr by this **Friday(6.14), 23:59**.

FYI, here are the statistics for this exam.
Average: 37
Standard Deviation: 22

**Comment by leporia** [comment]
On Gradescope I don't see the option for "Regrade request" for the final

**Reply by goodtaeeun** [reply]
Oh, my mistake. Please check again. I have enabled it.

**Comment by 33577** [comment]
안녕하세요
한 학기 동안 감사했습니다 :) 
Klms의 성적 항목별 가중치가 올바른 값인지 여쭤보고 싶습니다
지금 나온대로 기말 시험이 과제 하나와 똑같은 가중치인 게 맞나요?
감사합니다!

I wonder whether current weights in klms's grade is valid or not.

**Reply by goodtaeeun** [reply]
현재 KLMS의 성적은 Gradescope의 과제와 연동해두었기 때문에 원점수 그대로만 들어가고 있습니다.
참고로 각 과제별 가중치는 동일하며 총점의 50%를 차지하고, 기말고사는 40%를 차지합니다. 나머지 10%는 보너스 점수의 영역입니다.

---

The scores in the KLMS are the raw scores of the homework and the exam since they are directly linked to the Gradescope system. FYI, scores of all the homework is equally weighted and take up 50% of the final grade. The final exam takes 40% and the remaining 10% can be obtained with bonus points.
